logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/AuthenticationScope.kt

```diff
@@ -9,10 +11,17 @@ import com.wire.kalium.logic.data.auth.login.SSOLoginRepositoryImpl
 import com.wire.kalium.logic.data.register.RegisterAccountDataSource
 import com.wire.kalium.logic.data.register.RegisterAccountRepository
 import com.wire.kalium.logic.di.MapperProvider
+import com.wire.kalium.logic.feature.appVersioning.CheckIfUpdateRequiredUseCase
+import com.wire.kalium.logic.feature.appVersioning.CheckIfUpdateRequiredUseCaseImpl
 import com.wire.kalium.logic.feature.auth.sso.SSOLoginScope
 import com.wire.kalium.logic.feature.register.RegisterScope
 import com.wire.kalium.network.networkContainer.UnauthenticatedNetworkContainer
 
+class AuthenticationScopeProvider(private val clientLabel: String) {
```

Comment:
this can be a companion object inside AuthenticationScope

```kotlin
class AuthenticationScope {
    companion object {
        fun create(....): AuthenticationScope = AuthenticationScope(....)
    }
}
```

Reply:
but in that case we couldn't provide it in `GlobalKaliumScope`

Reply:
Is it okay for `AuthenticationScope` to not be a singleton? Because right now `AuthenticationScopeProvider` creates a new one every time

Reply:
Is it okay for AuthenticationScope to not be a singleton? Because right now AuthenticationScopeProvider creates a new one every time